merlin-server/sql/merlin/plan_merge.sql

```diff
@@ -0,0 +1,619 @@
+create type activity_change_type as enum ('none', 'add', 'delete','modify');
+create type conflict_resolution as enum ('none','supplying', 'receiving');
+
+/*
+Stores all of the activities associated with a merge
+Gets added to by:
+    - taking the diff of the supplier and receiver and uploading all non-conflicting changes with relevant tag
+      -  (conflicting ids go into the conflicting activity tables)
+    - uploading a resolved conflict with either the tag 'delete' or 'modify' depending on context
+      -  (none and add cannot cause conflict)
+      -  (a new version of the conflict resolution will replace the version currently in the table
+    - deletes are placed as the full activity, but are 'marked to delete' essentially
+-- Because of this, to get the full list of activities involved in an in-progress merge requires querying here and conflicting_activities
+ */
+-- TODO: Consider removing some of the defaults from this table? Decision depends on if we can have an activity
+--        that doesn't have a field in this area somehow.
+create table merge_staging_area(
+    request_id integer
```

Comment:
Can we rename this to `merge_request_id` to be consistent with the `merge_request_id` column in the `conflicting_activities` table?

Reply:
Done